.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
add this file to your .gitignore file

Reply:
Okay, however, I'm still unclear about what it will actually do.